numpy/f2py/func2subr.py

```diff
@@ -164,7 +164,10 @@ def add(line, ret=ret):
             pass
         else:
             add('interface')
-            add(rout['saved_interface'].lstrip())
+            saved_interface = rout['saved_interface'].lstrip().split('\n')
+            for line in saved_interface:
+                if not ('use' in line and '__user__' in line):
```

Comment:
Assuming line is a str and not a list: the second condition is a subset of the first one, and so is redundant - was that deliberate?

Reply:
@eric-wieser thanks for the quick review! I have changed the condition to something more meaningful. Actually it should remove everything that looks like `use something__user__stuff`.